packages/paste-website/src/components/DoDont.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+const StyledWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-column-gap: 24px;
+`;
+
+interface DoDontProps {
+  children?: React.ReactNode;
+}
+
+function DoDont(props: DoDontProps) {
+  return <StyledWrapper>{props.children}</StyledWrapper>;
+}
+
+const StyledChild = styled.div``;
+
+const StyledChildImg: React.FC<ChildProps> = styled.div`
+  margin-bottom: 16px;
+  border: 1px solid #ccd2dc;
+  border-bottom: ${(props: ChildProps) => (props.do ? '2px solid #23bf6e' : '2px solid #ce241a')};
+`;
+
+const StyledChildContent = styled.div``;
+
+const StyledChildTitle = styled.h5`
```

Comment:
Is there an opportunity to use the h5 component from the mdx content? Are they the same visually?

Reply:
I ended up using the `<Text>` component and setting some slight visual adjustments with props. Check it out and let me know if that looks ok.

Reply:
Nice! Looks perfect